app/assets/stylesheets/application.scss

```diff
@@ -57,3 +57,14 @@
   color: govuk-colour("dark-grey");
   font-style: italic;
 }
+
+// Show spaces in denylist entries using a special character to make them more obvious
+.app-denylist-entry-visible-space {
+  font-size: 0;
+
+  &::before {
+    content: "⎵";
```

Comment:
I had no idea that you could do this! I'm not sure how I feel about it, but it solved the problem.